plugins/jira-dashboard-backend/src/queries.ts

```diff
@@ -18,7 +18,8 @@ export const jqlQueryBuilder = ({
   components,
   query,
 }: JqlQueryBuilderArgs) => {
-  let jql = `project in (${project})`;
+  const projectList = Array.isArray(project) ? project : [project];
```

Comment:
Since it's always project[0] that is passed into this function, it will never be an array. See [this code](https://github.com/AxisCommunications/backstage-plugins/blob/main/plugins/jira-dashboard-backend/src/service/router.ts#L182). So it's not working for me. Any plan regarding this?

How will we handle this in frontend? Display multiple projects in the ProjectCard?